docs/guides/3_generating_proofs.md

```diff
@@ -0,0 +1,55 @@
+# Generating Proofs For Aligned
+
+## Table of contents
+
+- [SP1](#sp1)
```

Comment:
We should remove table of content. We don't have it in other pages and gibook provides a table of content automatically in the right side of the page

Reply:
removed